test/src/glossary/widgets/kana_list_test.dart

```diff
@@ -8,51 +10,93 @@ import '../../../helpers.dart';
 
 void main() {
   group("KanaList", () {
+    late final AppLocalizations l10n;
+    final List<Kana> kanaListSample = List.generate(107,
```

Comment:
Je vois qu'on génère 150 fois cette liste dans tous nos tests. Par curiosité, est-ce qu'il est facilement faisable d'avoir un fichier de test qui soit rempli de "Dummy Data" ? Si oui, est-ce qu'on voudrait implémenter ça pour éviter de recopier les déclarations de Kana, Kanji, Vocabulaire ... ?

Reply:
Ça pourrait être utile d'avoir un fichier avec ça par contre faudrait qu'on rajoute freezed pour les models (en + d'Isar, voilà un [exemple](https://medium.com/@masreplay/integrating-freezed-with-isar-for-effortless-data-management-in-flutter-df03c3e37e59)) comme ça on a pas besoin de faire des dummy data pour tout les cas de figure, on pourra facilement faire des `.copyWith` pour dupliquer. 

Je vais nous faire un ticket pour ça.

Reply:
Done! #125 et #124 !